index.d.ts

```diff
@@ -24,21 +24,14 @@ declare namespace mem {
 
```

Comment:
You need to update the description here too. Just use everything used in the readme.